frontend/public/components/node.tsx

```diff
@@ -149,50 +150,49 @@ const NodeGraphs = requirePrometheus(({node}) => {
   const ipQuery = nodeIp && `{instance=~'${nodeIp.address}:.*'}`;
 
   return <React.Fragment>
-    <div className="row">
-      <div className="col-md-12 col-lg-4">
+    <Grid gutter="md">
+      <GridItem lg={4} md={12}>
```

Comment:
If we change this here, we should do it for the other charts on the pod and namespace pages.

Are these breakpoints really the same? I thought Patternfly had different sm/md/lg/xl breakpoints than Bootstrap. We were trying to avoid using more than one grid system to avoid confusion around that.

cc @rhamilto

Reply:
> We were trying to avoid using more than one grid system to avoid confusion around that.

+1

Reply:
Ah, no the breakpoints are not the same, but they can be mapped to equivalent ones:

xs = Bootstrap 3 xs
sm has no equivalent in Bootstrap 3
md = Bootsrap 3 sm
lg = Bootstrap 3 md
xl = Bootstrap 3 lg
xl2 has no Bootstrap equivalent